src/app/shared/database-service.ts

```diff
@@ -563,6 +565,36 @@ class DatabaseService {
     }
   }
 
+  /**
+   * Retrieves all available value sets in Query Connector.
+   * @returns A list of value sets registered in the query connector.
+   */
+  static async getAllValueSets(): Promise<QCResponse<DibbsValueSet>> {
```

Comment:
think we can use the `@adminRequired` decorator here since this is in the service class